### DISCUSSION TITLE
Why i didn't get sponsor button in my profile

### DISCUSSION THREAD

**Discussion by Nicat-dcw** [community/community#57161] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Two months ago i submitted my github sponsorship form but i didn't get email from github support. My @Greesybot organization approved for github sponsors button

**Comment by anjuan** [comment]
Hi @Nicat-dcw. Are you trying to set up an individual sponsorship for yourself? Meaning, you want a sponsorship button to show here: https://github.com/Nicat-dcw

**Comment by anjuan** [comment, marked as answer]
Hi again, @Nicat-dcw. Our internal team approved your sponsors profile. So, you should be good to go!

**Reply by Nicat-dcw** [reply]
Thanks

**Comment by zkromeo** [comment]
Hi,
I submitted my application for the GitHub Sponsors program about a month ago, but my profile still shows the status as "Pending." The "Sponsor" button is not visible on my profile, and it only shows "Pending" with no updates.
Please look into this issue or let me know if any additional steps are needed.
This is my GitHub 
https://github.com/zkromeo
Thanks
